storage/src/main/scala/coop/rchain/storage/Bb.scala

```diff
@@ -0,0 +1,117 @@
+/*               __          __                                         *\
+**    __________/ /_  ____ _/_/___                                      **
+**   / ___/ ___/ __ \/ __ `/ / __ \     RChain API                      **
+**  / /  / /__/ / / / /_/ / / / / /     (c) http://rchain.coop          **
+** /_/   \___/_/ /_/\____/_/_/ /_/                                      **
+\*                                                                      */
+
+// Bb wraps Java ByteBuffer and adds some type genericity.
+// Methods that translate a ByteBuffer to a Scala type
+// resist type genericity due to ByteBuffer's lack of that.
+
+package coop.rchain.storage
+
+import java.nio.ByteBuffer
+import java.nio.ByteBuffer.allocateDirect
+import java.nio.charset.StandardCharsets.UTF_8
+
+object Bb {
+
+  def create[T](value: T): Option[ByteBuffer] = {
+    if (value.isInstanceOf[Short]) {
```

Comment:
Use match/case instead of if: https://stackoverflow.com/questions/5177180/can-we-use-match-to-check-the-type-of-a-class